Can't get it running

I always get this error message and the PR isn't created at all. Could you give me a hint what's going wrong? 

Here's the log file... (head branch name was edited for privacy reasons) 

```
2020-12-21T20:28:18.0083431Z ##[group]Run tgymnich/fork-sync@v1.3
2020-12-21T20:28:18.0084080Z with:
2020-12-21T20:28:18.0085256Z   github_token: ***
2020-12-21T20:28:18.0085780Z   base: fm-sys:master
2020-12-21T20:28:18.0086262Z   head: <name of original owner>:master
2020-12-21T20:28:18.0086806Z   pr_title: Update Fork
2020-12-21T20:28:18.0087264Z   merge_method: merge
2020-12-21T20:28:18.0087690Z   auto_approve: false
2020-12-21T20:28:18.0088143Z ##[endgroup]
2020-12-21T20:32:20.6346723Z request failed after 4 retries with a delay of 60
2020-12-21T20:32:20.6356377Z ##[error]Failed to create or merge pull request: HttpError: Validation Failed: {"resource":"PullRequest","field":"base","code":"invalid"} 
```

--- a/README.md
+++ b/README.md
@@ -3,7 +3,7 @@
 ![Version](https://img.shields.io/github/v/release/tg908/fork-sync?style=flat-square)
 
 Github action to sync your Forks.
-This action uses octokit and the GitHub API to automatically create and merge a pull request with the head defined by `owner`:`head` into the base defined by `base`. If you create a PR in the same repository you can omit the `owner` parameter.
+This action uses octokit and the GitHub API to automatically create and merge a pull request with the head defined by `head` into the base defined by `base`. The head branch owner is defined by `owner`. If you create a PR in the same repository you can omit the `owner` parameter.
 
 # Example Workflow
 
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: tgymnich/fork-sync@v1.2
+      - uses: tgymnich/fork-sync@v1.3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           owner: llvm
@@ -53,7 +53,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: tgymnich/fork-sync@v1.2
+      - uses: tgymnich/fork-sync@v1.3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           owner: llvm
@@ -79,5 +79,7 @@ jobs:
 |   personal_token| ✅          |                        |   Usertoken for the user to auto approve the pull request   |
 
 ⚠️ $current_repo_owner is your own username!
+
+⚠️ Only provide the branch name for `head` and `base`. `user:branch` will not work! 
 
 ⚠️ * if `auto_approve` is set to `true` you must provide the `personal_token`! 

